database/drop_db_gcp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# 
```

Comment:
An "Are you sure Y/N :" might be worthwhile here

Reply:
done